instance/models/openedx_appserver.py

```diff
@@ -176,6 +176,10 @@ class Meta:
             "behind a firewall. (In the django admin, separate group names with a comma.)"
         )
     )
+    enable_vm_dns_records = models.BooleanField(
+        default=False,
+        help_text="Set additional DNS records pointing directly to the active app servers."
+    )
```

Comment:
@smarnach Why making it an option, especially defaulting to off? This seem like a useful feature for all our instances?